modules/govuk_jenkins/files/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy

```diff
@@ -311,6 +298,19 @@ def publishGem(String repository, String branch) {
     sh("gem build ${repository}.gemspec")
     sh("gem push ${repository}-${version}.gem")
   }
+
+  def taggedReleaseExists = sh(
+    script: "git ls-remote --exit-status --tags origin v${version}",
```

Comment:
Should this be `--exit-code` as per https://git-scm.com/docs/git-ls-remote.html? Doesn't work locally, at least on my Mac (Git 2.9.0)